lib/logging.js

```diff
@@ -8,7 +8,7 @@ let level;
 if (process.env.NODE_ENV === 'test') {
     level = bunyan.FATAL + 1; //turn off logging
 } else {
-    level = process.env.LOG_LEVEL || 'info';
```

Comment:
@ElBabbler that would help, apart from that I think 'info' was wrongly used in other pieces of code for `salior`, see here - https://github.com/elasticio/sailor-nodejs/pull/81/commits/2d6bf73746bb3b85456b304f9e9074d2465c57d6

Reply:
Done!

Reply:
@zubairov @ElBabbler can you confirm that everything has been fixed and ready to merge?